src/map/packets.h

```diff
@@ -3731,6 +3731,23 @@ packet(0x96e,-1,clif->ackmergeitems);
 	packet(0x0923,36,clif->pStoragePassword,0);
 	packet(0x0a27,8); // ZC_RECOVERY2
 	packet(0x09f7,75); // ZC_PROPERTY_HOMUN_2
+	packet(0x09E8,11,clif->rodex_open_mailbox); // CZ_OPEN_MAILBOX
```

Comment:
Search packet with correct size. Often size changed and this mean packet used for some thing else. You need only packets with size like in you pr